preview-docs/GraphRoles.md

```diff
@@ -2,20 +2,23 @@
+We recently [introduced user roles to Studio](https://www.apollographql.com/docs/studio/org/members/#member-roles) and are now tackling Phase 2 of the [roles and permissions plan](https://github.com/apollographql/apollo-studio-community/blob/main/preview-docs/UserRoles.md), which involves introducing access control to graphs.
 
 We've identified new requirements for access control at the graph level since the first doc was written, including needs to control permissions with variant granularity.
 
-Here's what we plan to do for the next phase of work in Studio on access control, which we hope will cover your needs for graph and variant permissions. If this plan does not address your needs, please let us know so we can understand what the gaps are.
-
-1. Introduce a **Graph Admin** role that can operate with all permissions of the **Contributor** role, but can also manage user access to the graph and delete the graph. (Our current **Admin** role will be renamed to **Org Admin**. **Graph Admin**s and **Org Admin**s will have identical permissions to act on graphs, but org admins will also be able to manage org membership, billing, etc.)  The **Contributor** role will be made less powerful. It will not be able to perform management tasks on the graph like updating integrations and settings; it will have all the permissions of **Observer** plus the ability to push new versions of the graph's schema. (When this role is added, we may automatically change existing **Contributor**s to **Graph Admin**s, or leave them as **Contributor**s and let org admins upgrade them if that's useful. Let us know if you have a strong preference!)
+1. Introduce a **Graph Admin** role that can operate with all permissions of the **Contributor** role, but can also manage user access to the graph and delete the graph. (Our current **Admin** role will be renamed to **Org Admin**. **Graph Admin**s and **Org Admin**s will have identical permissions to act on graphs, but org admins will also be able to manage org membership, billing, etc.)  The **Contributor** role will be made less powerful. It will not be able to perform management tasks on the graph like updating integrations and settings; it will have all the permissions of **Observer** plus the ability to push new versions of the graph's schema.
 2. Introduce a way to override any role a user might have in the org with a higher role for a specific graph. A user might be an **Observer** in the org, but can be promoted to a **Contributor** on a specific graph. When users create a graph, they are automatically made **Graph Admin** on that graph. **Graph Admin**s and **Org Admin**s can update these graph-specific permissions.
 3. Introduce a way to mark variants as **protected**. On a protected variant, you need to be a **Graph Admin** rather than a **Contributor** to push new versions of the graph's schema to that variant. **Graph Admin**s and **Org Admin**s can control whether variants are protected.
```

Comment:
```suggestion
4. Introduce a way to mark variants as **protected**. On a protected variant, you need to be a **Graph Admin** rather than a **Contributor** to push new versions of the graph's schema to that variant. **Graph Admin**s and **Org Admin**s can control whether variants are protected.
```